pygridtools/core.py

```diff
@@ -642,9 +645,9 @@ def update_cell_mask(self, mask=None, merge_existing=True):
         masked.cell_mask = mask
         return masked
 
-    def mask_nodes(self, polyverts, min_nodes=3, inside=False, use_existing=False,
-                   triangles=False):
-        """ Create mask the ModelGrid based on its nodes with a polygon .
+    def mask_nodes(self, gdf, min_nodes=3, inside=False,
```

Comment:
@ricaenriquez @lucashtnguyen 

Wanted y'all to see this. Now instead of passing lists of coordinates, you can just pass a GeoDataFrame of (Multi)Polygons.

This will break old grid generating scripts that you have.